src/blocks/custom_dbus.rs

```diff
@@ -31,11 +31,18 @@
 //! busctl --user call rs.i3status /my_path rs.i3status.custom SetState s good
 //! ```
 //!
+//! Because it's impossible to publish objects to the same interface from different
+//! processes, having multiple dbus blocks in different bars won't work. As a workaround,
+//! you can set the env var `I3STATUS_DBUS_INTERFACE` to set the interface a bar works on to
```

Comment:
Probably better to call it `I3RS_DUBS_INTERFACE` because it is shorter and this is not `i3status`. Also this will be similar to [github's token env variable](https://github.com/greshake/i3status-rust/blob/85cd6cc30ce873b14712dbaa2cd4786d26136add/src/blocks/github.rs#L86).

Reply:
By the way, not sure but I think technically it is a _name_, not _interface_.

Reply:
Ok. I've changed it to I3RS_DBUS_NAME.